src/main/java/seedu/duke/Parser.java

```diff
@@ -153,6 +168,13 @@ private ArrayList<String> partitionCommandTypeAndDetails(String fullCommandDetai
         return processedCommandDetails;
     }
 
+    void checkForEmptyListParameters(String commandListDetails) throws MissingListDetailException {
```

Comment:
Would be good to include either public or private. Ambiguous if left empty

Reply:
private will be good